packages/DropdownMenu/stories/examples/Showcase.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { Rule, Tagline } from "storybook/assets/styles/common.styles";
-import { select } from "@storybook/addon-knobs";
+import { select, text } from "@storybook/addon-knobs";
+import { AlignTypes } from "@paprika/helpers/lib/customPropTypes";
```

Comment:
`out of the scope for this PR`

I think we should start creating a types file on helper package and have something like:

```
// types.js
export default {
  left: "LEFT",
 right: "RIGHT",
 xsmall: "XSMALL",
 large: "LARGE",
 ...etc
}
```

```js
import types from "@paprika/helpers/lib/types";

const defaultProps: {
  size: types.xsmall
}

function Something(props) {
   const {size} = props;
   if (size === types.xsmall) {
        render ""xsmall"
   }

   return "something else"
}
```

cc @mikrotron  what do you think?

Reply:
Yeah, I think having a `types` packages is a good idea.  I don't think we should export the t-shirt sizes as the default export, but that's a small implementation detail.  

Another possibility is to have things like `types`, `helpers`, maybe even `tokens` and `stylers` all bundled together in a `core` package...

Reply:
shouldn't all of them be part of the helper package?